dsl/pace/dsl/stencil.py

```diff
@@ -895,15 +965,18 @@ def from_origin_domain(
         """
         if self.config.compare_to_numpy:
             cls: Type = CompareToNumpyStencil
+            extra_kwargs = {}
         else:
             cls = FrozenStencil
+            extra_kwargs = {"timing_collector": self.timing_collector}
```

Comment:
Why do we only collect timings if not comparing to numpy? These seem like orthogonal configuration options.